api/config/rolepermission.php

```diff
@@ -299,6 +293,48 @@
             'fr' => 'Suspendre ou débloquer sa propre candidature présentée',
         ],
 
+        'view-own-applicationNotes' => [
+            'en' => 'View the notes associated with my own Applications',
+            'fr' => 'Consulter les notes associés avec mes propres candidatures',
+        ],
```

Comment:
Accidental? Does not appear used nor can I think of a use case

Reply:
True, I was just following a pattern, but there's not really a use case